### PR TITLE
Send notification when master build fails

### PR DESCRIPTION
**Pull request by geigerzaehler**
Send notification when master build fails.

### CODE REVIEW

File: .buildkite/pipeline.yaml [hunk 1]
```diff
@@ -31,3 +31,8 @@ steps:
     agents:
       platform: "linux"
       production: "true"
+
+notify:
+  - email: "thomas@monadic.xyz"
```
Comment (NunoAlexandre) [inline .buildkite/pipeline.yaml:36]: Can it be sent to the team email?

Is there an integration with irc? It'd be cool to get a message in our channel.

Comment (CodeSandwich) [reply .buildkite/pipeline.yaml:36]: There's only integration with email and slack. Unless we can bridge an email address or a dummy slack account with IRC, we won't be getting notifications.

Comment (NunoAlexandre) [reply .buildkite/pipeline.yaml:36]: Maybe can plug something using https://ifttt.com/. Using the team email would suffice for now though.

Comment (geigerzaehler) [reply .buildkite/pipeline.yaml:36]: I don’t want to send it to `registry@monadic.xyz` to not spam the non-devs that are subscribed there. What do you think about creating a `registry-notification@monadic.xyz` group?

Comment (CodeSandwich) [reply .buildkite/pipeline.yaml:36]: Cool! Or `registry-devs`. Can we see all the existing groups somewhere? Maybe such a group already exists.

Comment (geigerzaehler) [reply .buildkite/pipeline.yaml]: Done

**Pull request merged**